ui/components/templates/components/Brandlogo.jinja

```diff
@@ -0,0 +1,9 @@
+{# def #}
+
+<!-- vsht logo -->
+                <img
+                  class="ui image vsht-logo"
```

Comment:
when we have a dedicated less component for this, we should use it (ui image brandlogo)